modules/UI/toolbars/ToolbarToggler.js

```diff
@@ -35,7 +35,6 @@ function hideToolbar(force) { // eslint-disable-line no-unused-vars
     toolbarTimeoutObject = null;
 
     if (Toolbar.isHovered()
-            || APP.UI.isRingOverlayVisible()
```

Comment:
Restored in https://github.com/jitsi/jitsi-meet/commit/8c426e8bb75bb467cadb17ab61193bf733636c2f